### ISSUE TITLE
Show 'Canvas / Group / Previous Sibling' Selection in 'Edit Constraints' Menu

### ISSUE THREAD

**Issue by brentcas** [matt-curtis/Fluid-for-Sketch#76]
Hey, I find it a little annoying to have to click into the dropdown to see what the sizing / positioning relationship is relative to.

**Comment by covall** [comment]
Yes - it'a a valid information and it would be nice if it was visible all the time.
